lib/plugins/inventory.js

```diff
@@ -382,7 +382,7 @@ function inject (bot, { version, hideErrors }) {
       if (item) {
         item.slot = slot
       }
-      bot.inventory.slots[slot] = item
+      if (!Item.equal(bot.inventory.slots[slot], item, true)) bot.inventory.updateSlot(slot, item)
```

Comment:
why ?

Reply:
Avoids emitting updateSlot events on slots that have stayed the same, which I don't consider an update